modules/adding-tls-termination.adoc

```diff
@@ -0,0 +1,60 @@
+// Module included in the following assemblies:
+//
+// * networking/aws_load_balancer_operator/add-tls-termination.adoc
+
+:_content-type: PROCEDURE
+[id="nw-adding-tls-termination_{context}"]
+= Adding the tls termination on the AWS Load Balancer
+
+You can route the traffic for the domain `example.com` to pods of a `service` `example-service`, and add TLS termination on the AWS Load Balancer.
+
+.Prerequisites
+
+.Procedure
+
+. Deploy the operator and create an instance of the resource `AWSLoadBalancerController`:
```

Comment:
Operator is always capitalized per https://github.com/openshift/openshift-docs/blob/main/contributing_to_docs/doc_guidelines.adoc#operator-name-capitalization

Reply:
Ack'ed